src/main/java/net/sf/jabref/gui/shared/OpenSharedDatabaseDialog.java

```diff
@@ -39,9 +41,16 @@
 import net.sf.jabref.shared.DBMSConnector;
 import net.sf.jabref.shared.DBMSType;
 import net.sf.jabref.shared.exception.DatabaseNotSupportedException;
+import net.sf.jabref.shared.prefs.SharedDatabasePreferences;
+import net.sf.jabref.shared.security.Password;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 
 public class OpenSharedDatabaseDialog extends JDialog {
 
+    private static final Log LOGGER = LogFactory.getLog(Password.class);
```

Comment:
Change the class in getLog to the actual class name (OpenSharedDatabaseDialog)
Otherwise logs shows wrong class.